build/configure-ct

```diff
@@ -39,5 +39,5 @@ then
       ;;
   esac
 
-  ./configure GMOCK_DIR=${GMOCK} ${CONFIGURE_FLAGS} PROTOC=${INSTALL_DIR}/bin/protoc PKG_CONFIG_PATH="${INSTALL_DIR}/lib/pkgconfig" CPPFLAGS="-I${INSTALL_DIR}/include" CXXFLAGS="${CXXFLAGS} ${EXTRA_CXXFLAGS}" LDFLAGS="-L${INSTALL_DIR}/lib $EXTRA_LDFLAGS" LIBS="${EXTRA_LIBS}" INSTALL_DIR=${INSTALL_DIR} || tail -1000 config.log
+  ./configure GMOCK_DIR=${GMOCK} ${CONFIGURE_FLAGS} PROTOC=${INSTALL_DIR}/bin/protoc PKG_CONFIG_PATH="${INSTALL_DIR}/lib/pkgconfig" CPPFLAGS="-I${INSTALL_DIR}/include" CXXFLAGS="${CXXFLAGS} ${EXTRA_CXXFLAGS}" LDFLAGS="-L${INSTALL_DIR}/lib $EXTRA_LDFLAGS" LIBS="${EXTRA_LIBS}" evhtp_CFLAGS="-I${INSTALL_DIR}/include/evhtp" evhtp_LIBS="-levhtp" INSTALL_DIR=${INSTALL_DIR} || tail -1000 config.log
```

Comment:
This is the value that pkg-config would have provided.  You can see here that pkg-config's cflags for evhtp will *always* end in the string `/evhtp`: https://github.com/ellzey/libevhtp/blob/develop/evhtp.pc.in#L3

Now, I don't know whether that directory is actually needed or not; you might well be right that this isn't the right value to pull the headers in.  But it is the value that pkg-config will provide in evhtp 1.2.11 and I don't see any reason to deviate from that value.

If evhtp's pkg-config is returning wrong values, maybe we need to submit something upstream to them?